packages/material-ui-system/src/index.spec.tsx

```diff
@@ -27,41 +27,41 @@ function composeTest() {
   styler({ color: 'test', spacing: 1 });
 }
 
-function cssTest() {
+function sxTest() {
   function styleFunction(props: { color?: string; spacing?: number; theme?: object }) {
     return {};
   }
 
-  const wideOrNarrowStyleFunction = css(styleFunction);
+  const wideOrNarrowStyleFunction = styleFunctionSx(styleFunction);
 
   // narrow
-  wideOrNarrowStyleFunction({ theme: {}, css: { color: 'blue', spacing: 2 } });
-  // wide, undesire: `css` is required, marking it as optional breaks system/basics/#css-property
-  wideOrNarrowStyleFunction({ theme: {}, color: 'blue', spacing: 2, css: {} });
+  wideOrNarrowStyleFunction({ theme: {}, sx: { color: 'blue', spacing: 2 } });
+  // wide, undesire: `sx` is required, marking it as optional breaks system/basics/#sx-property
```

Comment:
What does "undesire" mean here?

Reply:
Sorry, missed to address this :( This is no longer the case, the undesire was that even tough you combined the old `css` with some other props, the prop was required. As we need to support now both `css` & `sx` they are both relaxed, so this is no longer an issue. It is fixed and the description is altered. Let me know if that makes sense.